<launch>
	<node pkg="d_fall_pps" name="CentralManagerService" output="screen" type="CentralManagerService">
	</node>

	<node pkg="d_fall_pps" name="ViconDataPublisher" output="screen" type="ViconDataPublisher">
		<rosparam command="load" file="$(find d_fall_pps)/param/ViconConfig.yaml" />
	</node>

	<!-- When we have a GUI, this has to be adapted and included -->
	<node pkg="d_fall_pps" name="my_GUI" output="screen" type="my_GUI">
	</node>

	<!-- PARAMETER SERVICE -->
	<node pkg="d_fall_pps" name="ParameterService" output="screen" type="ParameterService">
		<param name="type" type="str" value="coordinator" />
		<param name="agentID" value="0" />
		<rosparam command="load" file="$(find d_fall_pps)/param/SafeController.yaml" ns="SafeController" />
		<rosparam command="load" file="$(find d_fall_pps)/param/DemoController.yaml" ns="CustomController" />
	</node>

</launch>
